projects/dendrites/permutedMNIST/experiments/centroid.py

```diff
@@ -145,10 +150,38 @@ class CentroidFigure1BExperiment(CentroidFigure1B,
     ),
 )
 
+# Synaptic Intelligence + Dendrites on 10 permutedMNIST tasks
+SI_CENTROID_10 = deepcopy(CENTROID_10)
+SI_CENTROID_10["model_args"].update(hidden_sizes=[2000, 2000])
+SI_CENTROID_10.update(
+    experiment_class=SICentroidExperiment,
+    epochs=20,  # Note that SI only works well with ~20 epochs of training per task
```

Comment:
So here to make it comparable to what we already have should we also re-run some analysis of XdG, XdG+dendrite and dendrite only with 20 epochs to show that any difference is not also due to the number of epochs ? That would be best I think but what do you think ?

Reply:
I assume you mean SI instead of XdG, right? The number of epochs is just a hyperparameter, so I wouldn't worry too much about it. Also, they use 20 epochs in the XdG paper, too.